lighthouse-core/audits/errors-in-console.js

```diff
@@ -45,10 +45,9 @@ class ErrorLogs extends Audit {
 
   /** @return {AuditOptions} */
   static defaultOptions() {
-    return {};
+    return {ignoredPatterns: ['ERR_BLOCKED_BY_CLIENT.Inspector']};
```

Comment:
My initial reaction was `.Inspector` might be too narrow since I didn't remember it *always* having that suffix, but I couldn't find reports of it without it, so I might be making that up. Calling it out in case you had the same initial thought and were on the fence for another nudge :)

Reply:
There's a bunch of other blockedReasons that are fairly legit, so i think preserving those results seems good.  https://chromedevtools.github.io/devtools-protocol/tot/Network/#type-BlockedReason

Reply:
Ya for sure, but those would be caught by `{ignoredPatterns: ['ERR_BLOCKED_BY_CLIENT']}`? I was questioning if the `.Inspector` part needs to be here.